api/src/test/java/uk/gov/dhsc/htbhf/claimant/communications/PaymentCycleNotificationHandlerTest.java

```diff
@@ -0,0 +1,112 @@
+    }
+
+    @Disabled("HTBHF-2028")
+    @Test
+    public void shouldSendNewChildFromPregnancyEmail() {
+        Claim claim = aClaimWithExpectedDeliveryDate(LocalDate.now().minusWeeks(8));
+        PaymentCycleVoucherEntitlement voucherEntitlement =
+                aPaymentCycleVoucherEntitlementWithBackdatedVouchersForYoungestChild(LocalDate.now(), asList(LocalDate.now().minusWeeks(6)));
+        PaymentCycle paymentCycle = aPaymentCycleWithCycleEntitlementAndClaim(voucherEntitlement, claim);
+
+        paymentCycleNotificationHandler.sendNotificationEmails(paymentCycle);
+
+        verifyNewChildFromPregnancyEmailSent(paymentCycle, claim);
+        verify(upcomingBirthdayEmailHandler).handleUpcomingBirthdayEmails(paymentCycle);
+    }
+
+    private void verifyPaymentEmailNotificationSent(PaymentCycle paymentCycle, Claim claim) {
+        ArgumentCaptor<MessagePayload> payloadCaptor = ArgumentCaptor.forClass(MessagePayload.class);
+        verify(messageQueueClient).sendMessage(payloadCaptor.capture(), eq(MessageType.SEND_EMAIL));
+
+        assertThat(payloadCaptor.getAllValues()).hasSize(1);
+        assertThat(payloadCaptor.getValue()).isInstanceOf(EmailMessagePayload.class);
+        verifyPaymentEmail(paymentCycle, claim, payloadCaptor.getValue());
+    }
+
+    private void verifyNewChildFromPregnancyEmailSent(PaymentCycle paymentCycle, Claim claim) {
+        ArgumentCaptor<MessagePayload> payloadCaptor = ArgumentCaptor.forClass(MessagePayload.class);
+        verify(messageQueueClient).sendMessage(payloadCaptor.capture(), eq(MessageType.SEND_EMAIL));
+
+        assertThat(payloadCaptor.getAllValues()).hasSize(1);
+        assertThat(payloadCaptor.getValue()).isInstanceOf(EmailMessagePayload.class);
+
+        EmailMessagePayload payload = (EmailMessagePayload) payloadCaptor.getValue();
+        assertThat(payload.getEmailType()).isEqualTo(EmailType.NEW_CHILD_FROM_PREGNANCY);
+        assertThat(payload.getClaimId()).isEqualTo(claim.getId());
+
+        PaymentCycleVoucherEntitlement voucherEntitlement = paymentCycle.getVoucherEntitlement();
+        assertThat(payload.getEmailPersonalisation()).contains(
+                entry("First_name", claim.getClaimant().getFirstName()),
+                entry("Last_name", claim.getClaimant().getLastName()),
+                entry("payment_amount", formatVoucherAmount(paymentCycle.getTotalVouchers())),
+                entry("pregnancy_payment", ""),
+                entry("children_under_1_payment",
+                        "\n* " + formatVoucherAmount(voucherEntitlement.getVouchersForChildrenUnderOne()) + " for children under 1"),
+                entry("children_under_4_payment", ""),
+                entry("backdated_payment", formatVoucherAmount(voucherEntitlement.getBackdatedVouchers())),
+                entry("next_payment_date", EMAIL_DATE_FORMATTER.format(paymentCycle.getCycleEndDate().plusDays(1)))
+        );
```

Comment:
Are we going for these assertions being in individual tests rather than in `EmailAssertions`? My thought was that if they were in `EmailAssertions` then we might be able to refactor/reuse more easily than if they were scattered around the codebase.